### DISCUSSION TITLE
In French cuisine, “agneau” refers to which meat?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#549] [Galaxy Trivia] [answered]
**Bonus:** What are the most common cuisines around the world that use this meat?

**Comment by thuongtruong1009** [comment, marked as answer]
In French cuisine, "agneau" refers to lamb meat.

Lamb is a widely consumed meat in many cuisines around the world. Some of the most common cuisines that use lamb include:

Middle Eastern cuisine: Lamb is a staple in many Middle Eastern dishes, such as shawarma, kofta, and lamb chops.
Mediterranean cuisine: Lamb is a popular meat in Mediterranean cuisine, particularly in Greek and Turkish cuisine. Dishes such as gyro, moussaka, and lamb kebabs are often made with lamb.
Indian cuisine: Lamb is commonly used in Indian cuisine, particularly in the northern regions of the country. Dishes such as rogan josh, biryani, and keema are often made with lamb.
British cuisine: Lamb is a traditional meat in British cuisine, particularly in dishes such as roast lamb and shepherd's pie.
